udc/src/main/java/io/crate/udc/ping/PingTask.java

```diff
@@ -93,6 +97,10 @@ public Boolean isMasterNode() {
         return clusterService.localNode().isMasterNode();
     }
 
+    public String isEnterprise() {
```

Comment:
I think this could be package private, and maybe annotate with visibleForTesting